src/Network/Receive/ServerType0.pm

```diff
@@ -1105,7 +1105,9 @@ sub buy_result {
 	} else {
 		error TF("Buy failed (failure code %s).\n", $args->{fail});
 	}
-	if (AI::is("buyAuto")) {
+	if ($args->{fail} == 2){
+		AI::args->{recv_buy_packet_owg} = 1;
```

Comment:
in my opinion, it would be good to put a new config here, to queue autosell after buyAuto failure only if person wants it.
like: `buyAuto_onFail_sell` or something